lib/la_maquina/dependency_map/yaml_map.rb

```diff
@@ -5,6 +5,7 @@ def find(*args)
         submap = map
         args.each do |key|
           submap = submap[key]
+          break if submap.blank?
```

Comment:
because with the map of 

```yml
  object:
    subobject:
      - crucially_important_mapping
```
`map.find :object, :subobject, :typo` will return `subobject` which i am not convinced is correct.